### PR TITLE
Feature/reviewer management

### PR DESCRIPTION
**Pull request by Parathantl**

### CODE REVIEW

**Review by RaveenaWickramasinghe — DISMISSED** [review]
LGTM

File: src/routes/pageRoutes.js [hunk 1]
```diff
@@ -17,6 +18,9 @@ function PageRoutes() {
       <Navbar />
         <section className="content">
           <Switch>
+            <Route path="/signup" component={SignUp} />
```
Comment (rusiruavb) [inline src/routes/pageRoutes.js]: @Parathantl login and signup routes are already in the page routes. Please correct it

**Review by rusiruavb — CHANGES_REQUESTED** [review]

**Review by rusiruavb — DISMISSED** [review]
LGTM. Nice @Parathantl

**Review by rusiruavb — APPROVED** [review]
LGTM. Great work @Parathantl

**Pull request merged**